src/other/ReplayPage.jsx

```diff
@@ -161,6 +163,14 @@ const ReplayPage = () => {
     window.location.assign(`/api/positions/kml?${query.toString()}`);
   };
 
+  useEffect(() => {
+    const { from, to, deviceId } = state;
+    if (from && to && deviceId) {
+      setShowFilter(false);
+      handleSubmit({ from, to, deviceId });
```

Comment:
I don't think this is right. It basically ends up in inconsistent state.

Reply:
Hm, does it? `handleSubmit` seems to persist everything it needs for it to make sense https://github.com/traccar/traccar-web/blob/fcf357ce628986d7b137a3e3d6de41f014241ca7/src/other/ReplayPage.jsx#L136-L138 The download button for example behaves correctly.

Or do you mean something else?

Reply:
But not in the filter.

Reply:
The filter is not rendered in this case. Having it update in the filter would throw away your previously selected timerange on the trips report when you navigate back which feels very bad

Reply:
That is the inconsistency I'm talking about. You're basically just hiding it.

Reply:
OK, I moved the state handling into the report component itself.

It simply prefers the state filter but reverts back to the store when the user closes the replay controls. So no more hiding the filter, the user can change it if they decide to close the replay.

It also should make it possible to load other reports on page load with relatively few changes in the future, if there is a usecase for that.